docs/intro.rst

```diff
@@ -0,0 +1,103 @@
+Camus
+================
+
+Camus is a pipeline from Kafka to HDFS.
+It is a MapReduce job that does distributed data loading from Kafka to HDFS.
+At LinkedIn, Camus is used to process tens of billions of messages per day.
+The confluent Camus distribution supports the latest CDH distribution.
+
+Camus includes the following features:
+    #. Automatic topic discovery
+    #. Avro schema management
+    #. Date partitioning
+
+All work is done within a single MapReduce job divided into three stages:
+    #. Setup stage:
+        Fetch available topics and partitions from Zookeeper and latest offsets from Kafka.
+    #. Hadoop stage:
+        Allocates topic pulls among a set number of MapReduce tasks. Each task does the following:
+            #. Fetch events from Kafka server and collect count statistics.
+            #. Write data to corresponding directories based on timestamps of events.
+            #. Produce count events and write to HDFS.
+            #. Store updated offsets in HDFS.
+    #. Cleanup stage:
+        Read counts from all tasks, aggregates the values,
+        and submits the results to Kafka for consumption by other systems.
+
+Quickstart 
+----------
+
+Assuming you have confluent platform installed.
```

Comment:
Quickstart should focus on being able to set up a simple Camus workflow. Maybe move the start up of all dependent services to the installation section and keep quick start for setting up that simple workflow. Also include the steps to setup Hadoop locally or point to a very simple Hadoop quick start. The point of the quick start is to be able to setup a simple workflow locally so it should not assume anything is already setup.